### ISSUE TITLE
Guzzle panel crashes when Twig can't parse an XML with a dodgy namespace

### ISSUE THREAD

**Issue by AlexSit** [csarrazi/CsaGuzzleBundle#229]
Hi, 

The Guzzle panel in my symfony profiler crashes when Twig stumbles upon parsing a dodgy XML.

I've got this error message from my logs: 

```
{
	"message": "An exception has been thrown during the rendering of a template (\"Notice: DOMDocument::loadXML(): xmlns: URI randomNamespace is not absolute in Entity, line: 3\").",
	"context": {
		"exception": {
			"class": "Twig_Error_Runtime",
			"message": "An exception has been thrown during the rendering of a template (\"Notice: DOMDocument::loadXML(): xmlns: URI randomNamespace is not absolute in Entity, line: 3\").",
			"code": 0,
			"file": "/application/vendor/csa/guzzle-bundle/src/Resources/views/Calls/list.html.twig:38",
			"previous": {
				"class": "ErrorException",
				"message": "Notice: DOMDocument::loadXML(): xmlns: URI randomNamespace is not absolute in Entity, line: 3",
				"code": 0,
				"file": "/application/vendor/csa/guzzle-bundle/src/Twig/Extension/GuzzleExtension.php:60"
			}
		}
	},
	"level": 400,
	"level_name": "ERROR",
	"channel": "app",
	"datetime": {
		"date": "2018-05-16 10:51:53.542573",
		"timezone_type": 3,
		"timezone": "UTC"
	},
	"extra": []
}
```

Here is my code building the XML:

```
$initString = <<<XML
<?xml version="1.0" encoding="UTF-8"?>
<root>
</root>
XML;

$payloadObject = new \SimpleXMLElement($initString);

$child = $payloadObject->addChild('leg');
$child->addAttribute('documentId', '1');

$subChild = $child->addChild('sportsLeg', null, 'randomNamespace');
```

Then this XML gets passed into Guzzle to send a request to some other web-server.
The request fails and I go to the symfony profiler to check my request, I click _latest_, 
then I click the _Guzzle_ link on the left panel and I see this:

<img width="1438" alt="screen shot 2018-05-16 at 12 35 59" src="https://user-images.githubusercontent.com/17311244/40114736-eac01498-5905-11e8-83c8-cc58ff414ddc.png">

**Comment by csarrazi** [comment]
I guess we should ignore warnings when loading the XML file, using `LIBXML_NOWARNING`. This should normally fix your issue.

I'll try to submit a fix soon.

Can you mention which version of the bundle you are using? That would be great, thanks! :)

**Comment by luispabon** [comment]
This is on v3 against symfony 4.0.x

**Comment by csarrazi** [comment]
Thanks for the answer!

I’ll see what I can do in the upcoming days/weeks :)

**Comment by csarrazi** [comment]
Fixed by #231
